manifests/k/Kitware/CMake/3.23.0-rc5/Kitware.CMake.installer.yaml

```diff
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Kitware.CMake
+PackageVersion: 3.23.0-rc5
```

Comment:
I don't think we can add the release candidate

Reply:
Thank you! Please don't add RCs. I already commented [here](https://github.com/microsoft/winget-pkgs/pull/45416#issuecomment-1054717464) on this.

Reply:
The way the winget-automation-bot works is it looks as the GitHub tags, if the Publisher properly marks them with Latest and Pre-Release it will work as expected, however the CMake developers mark every release as Latest (Stable) so wingetbot picks it up.
So in this case @vedantmgoyal2009 needs to tell the bot to ignore tags ending with -rc*

Reply:
I have now updated the automation to ignore tags ending with `-rc*`. Thank you for mentioning the old comment again, maybe I had missed it since the comment was made after the PR was merged.